packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -121,6 +126,9 @@ find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 %doc %{gem_docdir}
 
 %changelog
+* Wed May 22 2019 Eric D. Helms <ericdhelms@gmail.com>
```

Comment:
this is missing a version :)

Reply:
Odd, I used obal.

Reply:
I've seen this when it could not figure out the current version for some reason.